core/src/main/java/nl/nn/adapterframework/receivers/PullingListenerContainer.java

```diff
@@ -228,15 +231,20 @@ public void run() {
 								return;
 							}
 
+							if (inProcessStateManager!=null) {
+								if ((rawMessage = inProcessStateManager.changeProcessState(rawMessage, ProcessState.INPROCESS))==null) {
+									return;
+								}
+								if (txStatus!=null) {
+									txManager.commit(txStatus);
+									txStatus = txManager.getTransaction(txNew);
+								}
```

Comment:
Ik kon mij vaag herrinneren waarom het belangrijk is om de status committen en een nieuwe aan te maken maar ik ben het vergeten. Zou jij hier stiekem een regeltje commentaar bij kunnen zetten?